tests/unit/test_settings_system.py

```
@@ -0,0 +1,162 @@
+        assert resp.get_json()["success"] is True
+
+    def test_level_err_variant(self, client):
+        """level='err' routes to logger.error."""
+        resp = client.post(
+            "/settings/client_log",
+            json={"level": "err", "message": "error msg"},
+        )
+        assert resp.status_code == 200
+        assert resp.get_json()["success"] is True
+
+    def test_outer_exception(self, client, monkeypatch):
```

Ruff (0.15.7)

[warning] Unused method argument: `monkeypatch`

(ARG002)